configs/classification/efficientnet_b0_int8_accuracy_aware.yml

```diff
@@ -0,0 +1,94 @@
+
+lr_finder:
+  enable: True
+  mode: TPE
+  stop_after: False
+  num_epochs: 6
+  step: 0.001
+  smooth_f: 0.01
+  epochs_warmup: 1
+  path_to_savefig: 'lr_finder.jpg'
+  max_lr: 0.01
+  min_lr: 0.001
+  n_trials: 15
+
+model:
+  name: 'efficientnet_b0'
+  classification: True
+  pretrained: True
+  save_chkpt: True
+  feature_dim: 1280
+
+custom_datasets:
+  roots: ['CIFAR100/train', 'CIFAR100/val']
+  types: ['classification_image_folder', 'classification_image_folder']
+  names: ['CIFAR100_train', 'CIFAR100_val']
+
+data:
+  root: './'
+  sources: ['CIFAR100_train']
+  targets: ['CIFAR100_val']
+  height: 224
+  width: 224
+  norm_mean: [0.485, 0.456, 0.406]
+  norm_std: [0.229, 0.224, 0.225]
+  save_dir: 'output/efficient_b0/log'
+  workers: 6
+  transforms:
```

Comment:
we decide to use only random_flip + augmix (it already includes random_rotate)

Reply:
But it is original efficienet config just with nncf section